source/connect/connection-options.txt

```diff
@@ -18,4 +18,316 @@ Specify Connection Options
    :keywords: connection string, URI, server, Atlas, settings, configure
 
```

Comment:
Suggestion: If we change the ToC depth to 2 here we can have the Connection Options subsections appear in the ToC.